modules/ecs/locals.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  # The namespace for the ECS
+  namespace = "devops-ic-ecs"
+
+  ecr_tag = "${local.namespace}-app"
+
+  # Environment variables from other variables
+  environment_variables = toset([
+    { name = "AWS_REGION", value = var.region },
+    { name = "HEALTH_CHECK_PATH", value = var.health_check_path },
+    { name = "APP_HOST", value = var.app_host },
+    { name = "APP_PORT", value = var.app_port }
+  ])
+
+  container_vars = {
+    namespace                          = local.namespace
+    region                             = var.region
+    app_host                           = var.app_host
```

Comment:
do we need to define app_host? will it be used anywhere. I'm aware we defined it in the infra-template as well but it doesn't look like the variable is used.

Reply:
I removed APP_HOST and some redundant vars in 9c2c996 ✅